spec/factories/course_materials.rb

```diff
@@ -4,6 +4,11 @@
     folder
     sequence(:name) { |n| "Material #{n}" }
     sequence(:description) { |n| "Material Description #{n}" }
-    attachment
+
+    after(:build) do |material|
```

Comment:
I think this is unnecessarily verbose. If one owner is to one attachment, the API should be completely consistent with the old one.